tools/workspace-plugin/tsconfig.lib.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
Otherwise `chalk` can't be imported:

```
TS1259: Module '"/Users/olfedias/WebstormProjects/office-ui-fabric-react/node_modules/chalk/index"' can only be default-imported using the 'esModuleInterop' flag  index.d.ts(415, 1): This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
```